python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -784,8 +788,8 @@ def __init__(self):
             for attr, item in self._schema.allItems():
                 ## If an object is hidden behind a getter method we can't assign a parent or defvalue so don't bother - rcurrie
                 if item.getProperties()['getter'] is None:
-                    defVal = self._schema.getDefaultValue(attr)
-                    self.setNodeAttribute(attr, defVal)
+                    setattr(self, attr, self._schema.getDefaultValue(attr))
```

Comment:
setNodeAttribute changes the object parent https://github.com/ganga-devs/ganga/pull/345/files#diff-3a5f5ed88f274eb570dccbab85337abfR360 doesn't it? Isn't this still required here?